app/controllers/api/base_controller.rb

```diff
@@ -107,6 +107,15 @@ def initialize
     end
 
     before_action :parse_api_request, :log_api_request, :validate_api_request, :validate_api_action
+    before_action :validate_response_format, :only => [:show]
```

Comment:
In this PR as that's done in update case and we also return json for async deletes. Thanks.